metadata.rb

```diff
@@ -4,7 +4,7 @@
 license           'Apache-2.0'
 description       'Prepares a machine to be an Omnibus builder.'
 long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version           '5.6.1'
+version           '5.6.2'
```

Comment:
We auto-bump this and generate a changelog with some tooling we have so in the future skip the version bump